tests/beaker/tests/01_create_motd_content.rb

```diff
@@ -0,0 +1,23 @@
+require 'master_manipulator'
+
+test_name 'FM-3918 - C95873 - Create /etc/motd file with content parameter'
+
+pp = <<-EOS
+class { 'motd':
+  content => 'Hello, this is the /etc/motd file content generated by a beaker test\n',
+}
+include motd
+EOS
+
+step 'Inject "site.pp" on Master'
+site_pp = create_site_pp(master, :manifest => pp)
+inject_site_pp(master, get_site_pp_path(master), site_pp)
+
+step 'Run Puppet Agent to apply the above manifest'
+confine_block(:except, :roles => %w{master dashboard database}) do
+  agents.each do |agent|
+    on(agent, puppet('agent -t --environment production'), :acceptable_exit_codes => [0,2]) do |result|
+      assert_no_match(/Error:/, result.stderr, 'Unexpected error was detected!')
+    end
+  end
+end
```

Comment:
You fail to verify that the contents of the MOTD file are correct after the Puppet run.